jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenProjectProperties.java

```diff
@@ -68,7 +67,6 @@ static MavenProjectProperties getForProject(
       return new MavenProjectProperties(
           project,
           makeEventEmitter(log),
-          new MavenJibLogger(log),
```

Comment:
Can we remove `MavenJibLogger` and its interface `JibLogger`?

Reply:
I'll get to these eventually. Loggers are still passed around in `BuildConfiguration` for the timers, which I'll be working on updating in later PRs. There are still some loose ends in this PR, but I wanted to submit some changes before it got too big.